WORKSPACE

```diff
@@ -1,21 +1,44 @@
 workspace(name = "ts_protoc_gen")
 
+
 git_repository(
-    name = "build_bazel_rules_nodejs",
-    remote = "https://github.com/bazelbuild/rules_nodejs.git",
-    tag = "0.4.0",
+  name = "io_bazel_rules_go",
+  commit = "6bee898391a42971289a7989c0f459ab5a4a84dd",  # master as of May 10th, 2018
+  remote = "https://github.com/bazelbuild/rules_go.git",
 )
+load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
+go_rules_dependencies()
+go_register_toolchains()
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
+http_archive(
+  name = "io_bazel_rules_webtesting",
+  strip_prefix = "rules_webtesting-master",
+  urls = [
+    "https://github.com/bazelbuild/rules_webtesting/archive/master.tar.gz",
+    ],
+)
+load("@io_bazel_rules_webtesting//web:repositories.bzl", "browser_repositories", "web_test_repositories")
+web_test_repositories()
 
+git_repository(
+  name = "build_bazel_rules_nodejs",
+  remote = "https://github.com/bazelbuild/rules_nodejs.git",
+  tag = "0.9.1",
+)
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "yarn_install")
 node_repositories(package_json = ["//:package.json"])
 
-git_repository(
-    name = "build_bazel_rules_typescript",
-    remote = "https://github.com/bazelbuild/rules_typescript.git",
-    tag = "0.10.0",
+# installs our node_modules
+yarn_install(
```

Comment:
I'm a little confused by this rule as ts-protoc-gen does not use yarn to manage dependencies? I'm guessing this rule provides interoperability with npm?

Reply:
Just pushed the commit to use rules_nodejs' npm_install. Works perfectly! 😝